db.php

```diff
@@ -0,0 +1,22 @@
+<?php
```

Comment:
Is it worth adding this whole file for all docker deployments?
What is the value compared to a volume mount?

Reply:
More for reference.  The forums had some examples but nothing in detail.  You could just as.easily add a blurb on the docker hub page.  I don't think the actual file will be pulled into the docker container as it's not referenced in the Dockerfile.

Reply:
Ah ok, good.
If not inside the Docker image, it should not live inside this repository.
Probably better to suggest a documentation update if so, indeed.

Reply:
Would you like for me to make a stab at a documentation addition?  If so, how would you like to receive that?

100% your project!  Just glad to share my specific use case